src/ParcelRegistry.Api.CrabImport/CrabImport/IdempotentCommandHandlerModuleProcessor.cs

```diff
@@ -1,23 +1,23 @@
 namespace ParcelRegistry.Api.CrabImport.CrabImport
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Threading;
-    using System.Threading.Tasks;
     using Be.Vlaanderen.Basisregisters.AggregateSource;
     using Be.Vlaanderen.Basisregisters.CommandHandling;
-    using Be.Vlaanderen.Basisregisters.GrAr.Import.Api;
     using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Import.Api;
     using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
     using Parcel;
     using Parcel.Commands.Crab;
     using SqlStreamStore;
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
 
     public class IdempotentCommandHandlerModuleProcessor : IIdempotentCommandHandlerModuleProcessor
     {
         private readonly ConcurrentUnitOfWork _concurrentUnitOfWork;
         private readonly ParcelCommandHandlerModule _parcelCommandHandlerModule;
-        private readonly Func<IHasCrabProvenance, Parcel, Provenance> _provenanceFactory;
+        private readonly Func<IHasCrabProvenance, Parcel, Provenance> _provenanceFactory = new ParcelProvenanceFactory().CreateFrom;
```

Comment:
Not sure why this is done? Seems to be asking for trouble, since now we have a provenance factory in the ctor, which is passed to the ParcelCommandHandlerModule, and this one